lib/utils.ts

```diff
@@ -13,9 +13,13 @@
  * @param {string | Map<string, string> | Record<string, string>} text
  * @return {{ [key: string]: string }}
  */
-function toLanguageObject(text: string | Map<string, string> | Record<string, string>): Record<string, string> {
-  if (typeof text === "string") {
-    return { en: text };
+function toLanguageObject(
+  text: string | Map<string, string> | Record<string, string> | undefined | null
+): Record<string, string> | null | undefined {
```

Comment:
Returning `undefined` is not correct and changes the JS function signature:
- Original JSDoc definition: `@return {{[p: string]: string}|null}`
- Unit tests for undefined/null/empty input parameters check that `null` is returned, `undefined` may not be returned.
- The logic in this function does not return `undefined`